src/widget/wfindonmenu.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <QAction>
+#include <QMenu>
+#include <QUrlQuery>
+
+class Track;
+
+class WFindOnMenu : public QMenu {
+    Q_OBJECT
+  public:
+    explicit WFindOnMenu(
+            QWidget* parent = nullptr);
+    ~WFindOnMenu() override = default;
+
+    void openInBrowser(const QString& serviceUrl,
```

Comment:
Here you should add 
`public slots:`

Reply:
Thank you. I have added it in my last commit.